cookbooks/java/metadata.rb

```diff
@@ -45,9 +45,9 @@
 end
 
 depends 'apt'
-depends 'windows'
-depends 'homebrew'
+# depends 'windows'
```

Comment:
Along those lines it might be good not to squash all these commits together when they are merged so that someone reading the commit history can distinguish what came from another source vs. our local modifications.